common/data/dtp/dtp.ts

```diff
@@ -0,0 +1,18 @@
+﻿/* eslint-disable */
+
+import {DtpSeverityType} from "./dtpSeverityType";
+import {CanGetById} from "../base/canGetById";
+import {getDataJsonByUrl, getObjectsTotalCount, StrapiBaseUrl} from "../dataHelpers";
+
+
+export class Dtp extends CanGetById{
+    public override async getObject(id: string): Promise<any> {
+        return super.getObject(id, "/dtps");
+    }
+
+    public async getObjectsBySeverity(type: DtpSeverityType){
+        const totalCount = await getObjectsTotalCount("/dtps");
+        return (await getDataJsonByUrl(StrapiBaseUrl +
```

Comment:
Let's not mix `${string-concat}` and `+`

`${StrapiBaseUrl}/dtps?filters[severity][$eq]=${type}&populate=data,geometry&pagination[pageSize]=${totalCount}`